### DISCUSSION TITLE
Error while install new plugin

### DISCUSSION THREAD

**Discussion by niceit** [Eugeny/tabby#4411] [Q&A]
Hi, while installing a new plugin, I opened the dev tool and see this error, do you have any idea why? 
In addition, in test mode (clone and build) it has no error

<img width="800" alt="2021-08-13_12-19-08" src="https://user-images.githubusercontent.com/3917898/129308510-6687071a-3491-4590-a435-585ba7a5eab1.png">

**Comment by Eugeny** [comment]
Looks like the aws-sdk package includes some scripts that run after installation. The error is likely due to `node` being not in `$PATH`, but that's generally a bad idea, since the user might not have Node installed at all.

**Reply by niceit** [reply]
Thank you, but my machine has node installed, the Dev one has no issue, but the packaged Tabby download from release has issue,  I don't think it's relevant as Node is not installed

**Reply by Eugeny** [reply]
Right, but is Node on $PATH or just installed somewhere? Since the `postinstall` script in `aws-sdk` only says `node ...`, it'd have to be on $PATH for it to work

**Reply by niceit** [reply]
I resolved the issue @Eugeny, thanks for your advice!
